torch/fx/graph.py

```diff
@@ -236,10 +251,18 @@ def python_code(self, root_module: str) -> Tuple[str, str, List[str]]:
                 assert isinstance(node.target, str)
                 body.append(f'{node.name} = {_format_target(root_module, node.target)}\n')
                 continue
+            elif node.op == 'output':
```

Comment:
The separation of python code from the function has gotten weaker over time. We should consider just having python code generating the whole function, and that would allow us to just generate 'return out' as a string here without the special handling.